microcloud/cmd/microcloud/main_init_preseed.go

```diff
@@ -653,6 +685,47 @@ func (p *Preseed) Parse(s *service.Handler, bootstrap bool) (map[string]InitSyst
 		systems[peer] = system
 	}
 
+	// Configure Ceph networks.
+	infos := make([]mdns.ServerInfo, 0, len(systems))
+	for _, system := range systems {
+		infos = append(infos, system.ServerInfo)
+	}
+
+	cephInterfaces, err := lxd.GetCephInterfaces(context.Background(), bootstrap, infos)
```

Comment:
For a bit of efficiency, should we wrap this in a `if p.Ceph.PublicNetwork != "" || p.Ceph.InternalNetwork != "" || !bootstrap`?